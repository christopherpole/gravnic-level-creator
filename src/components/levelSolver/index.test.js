import React from 'react';
import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import LevelSolver from './index';

configure({ adapter: new Adapter() });

describe('The root layout', () => {
  it('Renders without exploding', () => {
    const levelSolver = shallow(<LevelSolver />);

    expect(levelSolver).toHaveLength(1);
  });

  it('Matches the current snapshot', () => {
    const levelSolver = shallow(<LevelSolver />);

    expect(toJson(levelSolver)).toMatchSnapshot();
  });
});
