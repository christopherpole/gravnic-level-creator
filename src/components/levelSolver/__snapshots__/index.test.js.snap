// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The root layout Matches the current snapshot 1`] = `
<styled.section>
  <p>
    Level Solver goes here
  </p>
</styled.section>
`;
